builds/jenkins-build-pack-website/Jenkinsfile

```diff
@@ -1211,7 +1213,11 @@ def buildWebsite(framework) {
 				sh "rm -rf dist"
 				sh "npm cache clean --force"
 				sh "npm install"
-				sh "ng build --output-path=dist"
+				if(angularVersion >= 6){
+					sh "ng build --output-path=dist"
+				} else {
+					sh "ng build --prod --output-path=dist"
```

Comment:
Why is this always `--prod`? Shouldn't' this be environment specific?

Reply:
the command with the --prod flag was already present in the develop branch when I was merging it with azure support branch. And the reason for two different commands is to support new websites created with angular templates which are upgraded to 7 and to also support older angular websites of version 2.